examples/ui/standard_widgets_observers.rs

```diff
@@ -6,7 +6,6 @@
 
 use bevy::{
```

Comment:
We really need to rethink this example, but I can do this in follow-up.

Reply:
My complaints are mostly that this example is very poorly motivated pedagogically, especially if we're using observers for behavior by default :)